<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkIGTLStreamingManagementWidgetControls</class>
 <widget class="QWidget" name="QmitkIGTLStreamingManagementWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QmitkIGTLMessageSourceSelectionWidget" name="messageSourceSelectionWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>The selected IGTL device source</string>
       </property>
       <property name="text">
        <string>Selected IGTL Message Source: </string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="selectedSourceLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&lt;none&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startStreamPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start Stream</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopStreamPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop Stream</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fpsLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>FPS:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fpsSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>150</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkIGTLMessageSourceSelectionWidget</class>
   <extends>QTextEdit</extends>
   <header>QmitkIGTLMessageSourceSelectionWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
